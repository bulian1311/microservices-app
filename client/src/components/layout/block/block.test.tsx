import * as React from "react";
import { render, RenderResult } from "@testing-library/react";
import Block from "./block.component";

let documentBody: RenderResult;

beforeEach(() => {
  documentBody = render(<Block>Test block</Block>);
});

test("Block matces snapshot.", () => {
  const { baseElement } = documentBody;
  expect(baseElement).toMatchSnapshot();
});

test("Block render child.", () => {
  expect(documentBody.getByText("Test block")).toBeInTheDocument();
});
