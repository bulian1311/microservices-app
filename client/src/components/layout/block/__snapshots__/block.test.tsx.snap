// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Block matces snapshot. 1`] = `
<body>
  <div>
    <div
      class="sc-bdfBwQ dNZxre"
    >
      Test block
    </div>
  </div>
</body>
`;
